spec/features/trainees/degrees/adding_degree_spec.rb

```diff
@@ -0,0 +1,46 @@
+require "rails_helper"
+
+RSpec.feature "Adding a degree" do
+  scenario "adding a degree" do
+    given_a_trainee_exists
+    when_i_visit_the_degree_page
+    and_i_select_a_degree_type
+    and_i_click_continue_button
```

Comment:
it seems that there's a final assertion missing here? Do we need to add a `expect(summary_page).to be_displayed(id: trainee.id)` assertion as a final step?

Reply:
discussed offline, this will be changed as there is another step which is pending